data/content_purpose_supergroups.md

```diff
@@ -0,0 +1,20 @@
+# Content Purpose Supergroups
+
+These are the grouping that the navigation team came up with
+as a result of an investigation done using the "Jobs to be Done" framework in Q4.
+
+There a 5 high level Content Purpose Supergroups with each group together some subgroups
```

Comment:
Typo: `a` should be `are`
Typo: `with` should be `which`

Reply:
Good spot.